scripts/k6/buffer-template.js

```diff
@@ -0,0 +1,60 @@
+import { sleep } from "k6";
+import * as common from "./common.js";
+import { post, del } from "./common.js";
+
+export const options = common.options;
+
+export function setup() {
+  common.setup();
+
+  const res = post("v1/receivers", {
+    id: "buffer-template",
+    name: "Buffer API Template Benchmark",
+    exports: [
+      {
+        exportId: "test-export",
+        name: "test-export",
+        mapping: {
+          tableId: "in.c-buffer-template.data",
+          columns: [
+            { type: "id", name: "id" },
+            {
+              type: "template",
+              name: "template",
+              template: {
+                language: "jsonnet",
+                undefinedValueStrategy: "null",
+                content: `Body.a.b+":"+Body.c.f.g`,
```

Comment:
Tenhle benchmark nefunguje, asi proto, že `content` je tady blbě. Asi ten `template` column bude potřeba ještě líp zdokumentovat v https://github.com/keboola/developers-docs/pull/233. Můžeš mě nasměrovat, kde najdu ty globals/funkce, který jsou v tom jsonnet definovaný?

Reply:
Je to tu, zdokumentuj to rovno prosim:
https://github.com/keboola/keboola-as-code/blob/8ea64241ab898c785f372e4f790841a66dbb660f/internal/pkg/service/buffer/store/model/column/column.go#L240-L245

V tomto E2E teste je to spravne:
https://github.com/keboola/keboola-as-code/blob/8ea64241ab898c785f372e4f790841a66dbb660f/test/api/buffer/import/import/001-create/request.json#L44-L45

To, ze to vrati `500` namiesto `400 Bad Request` mam na TODO liste.